src/Pos/Wallet/WalletMode.hs

```diff
@@ -153,13 +152,13 @@ instance MonadIO m => MonadTxHistory (WalletDB m) where
     getTxHistory = Tagged $ \addr _ -> do
         chain <- WS.getBestChain
         utxo <- WS.getOldestUtxo
-        res <- fmap (fst . fromMaybe (error "deriveAddrHistory: Nothing")) $
+        _ <- fmap (fst . fromMaybe (error "deriveAddrHistory: Nothing")) $
             runMaybeT $ flip runUtxoStateT utxo $
             deriveAddrHistory addr chain
-        pure undefined
+        pure $ error "undefined"
```

Comment:
I'd like a descriptive error message here – at the very least it should give

* module name
* function name
* “can't happen” (ideally – a reason why it won't ever be executed)